_posts/2022-04-01-this-year-in-monero.md

```diff
@@ -0,0 +1,223 @@
+
+Monero's blockchain size grew ***144%*** faster YoY.
+
+## Emissions
+
+***04/2020 through 04/2021:***
+
+- Monero's block subsidy (emission) created ***362,406.83XMR***.
+- There were ***262,588*** total blocks, for an average block subsidy of ***1.38XMR*** per block for the year.
+- Monero's emission per day dropped from ***~1,200XMR*** to ***~750XMR*** over the course of the year.
+
+<img src="/blog/assets/2021-04-18-this-year-in-monero/monero-7y-emissions.png" position="center" style="border-radius: 8px;">
+
+***04/2021 through 04/2022:***
+
+- Monero's block subsidy (emission) created ***237,822.80XMR***.
+- There were ***262,567*** total blocks, for an average block subsidy of ***0.90XMR*** per block for the year.
+- Monero's emission per day dropped from ***~803XMR*** to ***~458XMR*** over the course of the year.
+
+<img src="/blog/assets/2022-04-01-this-year-in-monero/monero-8y-emissions.png" position="center" style="border-radius: 8px;">
+
+### Year over Year Change
+
+Monero's inflation decreased ***34.3%*** YoY and is steadily approaching the [tail emission](https://localmonero.co/knowledge/monero-tail-emission) of ***0.6XMR*** per block!
+
+## Fees
+
+***04/2020 through 04/2021:***
+
+- Monero's participants spent ***1,884.77XMR*** on fees in a year.
+- Monero averaged ***5.16XMR*** in fees spent per day.
+- There were ***262,588*** total blocks, for an average total fees to miners of ***0.00717767XMR*** per block for the year.
+- There were ***5,868,096*** total transactions, for an average of ***0.000321189XMR*** per transaction.
+
+<img src="/blog/assets/2021-04-18-this-year-in-monero/monero-7y-fees.png" position="center" style="border-radius: 8px;">
+
+***04/2021 through 04/2022:***
+
+- Monero's participants spent ***1,510.77XMR*** on fees in a year.
+- Monero averaged ***4.12XMR*** in fees spent per day.
+- There were ***262,567*** total blocks, for an average total fees to miners of ***0.00575384XMR*** per block for the year.
+- There were ***9,087,482*** total transactions, for an average of ***0.000166247XMR*** per transaction.
+
+<img src="/blog/assets/2022-04-01-this-year-in-monero/monero-8y-fees.png" position="center" style="border-radius: 8px;">
+
+### Year over Year Growth
+
+Monero's fees spent decreased ***19.84%*** YoY while the average transaction fee dropped ***48.24%***
+
+## Mining
+
+***04/2020 through 04/2021:***
+
+- Monero's network hashrate increased from ***1.21GH/s*** to ***2.29GH/s*** over the year.
+- Monero broke ***100,000 miners*** for the first time in its network history!
+- Monero unique miners increased from ***37,380*** to ***102,163*** over the year.
+
+<img src="/blog/assets/2021-04-18-this-year-in-monero/monero-hashrate-2021.png" position="center" style="border-radius: 8px;">
+
+***04/2021 through 04/2022:***
+
+- Monero's network hashrate increased from ***2.29GH/s*** to ***3.51GH/s*** over the year.
+- Monero unique miners decreased from ***102,163*** to ***44,672*** over the year.
```

Comment:
Also remove this as it incorrectly shows a drastic drop in miners due to a reporting error.